app/models/concerns/permitted_queries.rb

```diff
@@ -41,12 +48,20 @@ def permitted_apos
   # FIXME: seems to include display logic
   # @return [Array<Array<String>>] Sorted array of pairs of strings, each pair like: ["Title (PID)", "PID"]
   def permitted_collections
-    q = 'objectType_ssim:collection AND !project_tag_ssim:"Hydrus" '
-    q += permitted_apos.map { |pid| "#{SolrDocument::FIELD_APO_ID}:\"info:fedora/#{pid}\"" }.join(' OR ') unless is_admin
+    q = if is_admin
+          '*:*'
+        elsif permitted_apos.empty?
+          '-id:[* TO *]'
```

Comment:
I'm not sure, but we've used this construct elsewhere. Honestly, the whole thing could use a refactor (why send the query if you're going to throw away the results?), but I'm nervous about doing too much with this.. lightly.. tested code.